eZ/Publish/Core/settings/storage_engines/cache.yml

```diff
@@ -75,6 +75,9 @@ services:
             - "%ezpublish.spi.persistence.cache.inmemory.content.enable%"
         tags: ['ez.spi.persistence.cache.inmemory']
 
+    ezpublish.spi.persistence.cache.tagGenerator:
```

Comment:
1. For rebranding purposes any new code referring to legacy names is forbidden, unless can't be avoided. If you're concerned about integrity, feel free to add it either at the end of the file or import separate file with new names
2. camelCase service names are deprecated in SF. See [current naming convention](https://symfony.com/doc/current/contributing/code/standards.html#service-naming-conventions).
3. You don't really need to define this service like that. This should consist of:
    - FQCN definition of the actual implementation
    - interface service declaration aliasing that implementation
    - autowiring which injects this service properly, or if you don't want that, inject interface. We always inject interface aliases instead of definitions! (or at least should...)